src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -54,6 +54,22 @@ protected internal static bool TryParseLogLevel(string value, out LogLevel level
 			}
 		}
 
+		protected List<string> ParseSanitizeFieldNames(ConfigurationKeyValue kv)
+		{
+			if (kv == null || string.IsNullOrEmpty((kv.Value))) return DefaultValues.SanitizeFieldNames;
```

Comment:
What if I don't want to use sanitization, e.g. during troubleshooting?

Reply:
Hmm... 1 option I see here is that we could allow `""`, so we'd only check for `kv == null` and in the `appsettings.json` you could add an empty string with `"SanitizeFieldNames": ""`.